marklogic-data-hub/src/main/resources/ml-modules/root/marklogic-data-hub-spark-connector/initializeJob.sjs

```diff
@@ -27,6 +27,18 @@ const jobs = new Jobs.Jobs(config);
 
 const id = sem.uuidString();
 
+function buildJobPermissions(config) {
```

Comment:
This needs a comment as to why it's here too - otherwise a developer is going to see it and think - why is this duplicated here!?!? So explain that we've duplicated it here because we need this to work on 5.2.x, but the function is different between 5.2 and > 5.2.

Reply:
Also, I think you can get rid of the "jobs" library dependency as a result?

Reply:
Done.